tests/CMakeLists.txt

```diff
@@ -35,14 +35,19 @@ find_package(GTest REQUIRED CONFIG QUIET)
 add_executable(vulkan_tools_tests)
 target_sources(vulkan_tools_tests PRIVATE
     main.cpp
+    test_common.h
     icd/mock_icd_tests.cpp
 )
 get_target_property(TEST_SOURCES vulkan_tools_tests SOURCES)
 source_group(TREE "${CMAKE_CURRENT_SOURCE_DIR}" FILES ${TEST_SOURCES})
 
-target_include_directories(vulkan_tools_tests PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
+target_include_directories(vulkan_tools_tests PRIVATE ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Do we need CMAKE_CURRENT_BINARY_DIR?

Reply:
The test_common.h header is located in the current source directory, while there is a subfolder for each thing to test (currently only Mock ICD). Since the mock_icd_tests.cpp are in that subfolder, it needs to be able to find the header located in the parent directory.

If there is a better way to do this, don't hesitate to mention it.

Reply:
It's fine. I have just seen CMAKE_CURRENT_BINARY_DIR copy pasted more than anything and was wondering if it could be removed.

Reply:
The CMAKE_CURRENT_BINARY_DIR is so that test code can access the built executables through a generated header that contains the paths to the exes. This capability is straight up copied from the loader FYI. Any deficiencies in the loader's implementation (like not refreshing after switching compilers, bitness, build-types, etc) would be present here.